app/src/main/java/ch/sdp/vibester/helper/ArtistIdentifier.kt

```diff
@@ -0,0 +1,10 @@
+//package ch.sdp.vibester.helper
+//
+//import ch.sdp.vibester.R
+//import ch.sdp.vibester.api.LastfmMethod
+//
+//class ArtistIdentifier {
+//    var method: String = LastfmMethod.BY_ARTIST.method
+//    var artist: String = "Imagine Dragons"
+//    var mode: Int = R.string.imagine_dragons
+//}
```

Comment:
Using /* */ could make the code look cleaner instead of using // each line. Also, will this not be used anymore or is it commented for other purposes?

Reply:
Yeah, this calss in unecessary now, will delete